### DISCUSSION TITLE
Core .Net ConnectionString to a SQLEXPRESS DB does not work

### DISCUSSION THREAD

**Discussion by pesnunj0** [serenity-is/Serenity#6440] [Help]
I recently translate my project from Asp .Net MVC to Core .Net I use several connections. With mysql no problem but with SQL Server Express I get Connection Timeout Expired.
ConnectionString is exactly the same in Web.config than Appsetting.json

 "Kairos": {
      "ConnectionString": "server=SRVZECNOM\\SQLEXPRESS; Database=KairosPro;Trusted_Connection=True;MultipleActiveResultSets=true; Integrated Security=false;user id=sa;password=mypassword",
      "ProviderName": "System.Data.SqlClient"
    },

Data connection with Visual Studio works.
I would appreciate any help or idea

**Comment by juliermefelix** [comment]
Try changing from Server to Data Source
"ConnectionString": "Data Source=SRVZECNOM\SQLEXPRESS;

usually use like this:
```
   "Default": {
      "ConnectionString": "Data Source=EST00800109\\SQLEXPRESS; Initial Catalog=TesteAPI_Default_v1; Integrated Security=True",
      "ProviderName": "System.Data.SqlClient"
    },
```

**Comment by pesnunj0** [comment]
Thank you,
Exactly the same. The first connection attempt I always get a timeout, on the second or third attempt, the connection is successful. It may be a database problem although with MVC I did not have this problem.
Thanks anyway
